package.json

```diff
@@ -147,17 +147,17 @@
     "storybook-addon-performance": "^0.17.1",
     "terser-webpack-plugin": "^5.1.1",
     "ts-loader": "^8.0.12",
-    "typescript": "4.4.4",
+    "typescript": "5.3.x",
```

Comment:
This is the only concerning change here to me. What are the consequences of upgrading TS this far? Does this mean consumers break on lesser versions? A lot of breaking changes between 4.4 and 5.3